scripts/compileAjvValidators.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable */
+
+const fs = require('fs')
+const path = require('path')
+const Ajv = require('ajv')
+const standaloneCode = require('ajv/dist/standalone').default
+const addFormats = require('ajv-formats')
+const schema = require('@uniswap/token-lists/dist/tokenlist.schema.json')
+
+const ajv = new Ajv({ code: { source: true, esm: true } })
```

Comment:
Why not write both schemas to one file (see [docs](https://ajv.js.org/standalone.html#generating-functions-s-for-multiple-schemas-using-the-js-library-es6-and-esm-exports))

Reply:
i was fighting with it because the schemas have the same name, so it didn't like using the same instance. imo not worth the additional time to make it work